full-stack-tests/presentation/src/learning-snippets/Content.test.ts

```diff
@@ -997,7 +890,196 @@ describe("Learning Snippets", () => {
         const field = getFieldByLabel(content!.descriptor.fields, "Code");
         expect(content!.contentSet[0].values[field.name]).to.eq("Default - View 2");
       });
+
+    });
+
+    describe("ContentRelatedInstances", () => {
+
+      let imodel: IModelConnection;
+
+      beforeEach(async () => {
+        await initialize();
+        imodel = await SnapshotConnection.openFile("assets/datasets/Properties_60InstancesWithUrl2.ibim");
+      });
+
+      afterEach(async () => {
+        await imodel.close();
+        await terminate();
+      });
+
+      it("uses `relationshipPaths` attribute", async () => {
+        // __PUBLISH_EXTRACT_START__ ContentRelatedInstances.RelationshipPaths.Ruleset
+        // The ruleset has a specification that returns `bis.ModelSelector` content instances and a specification
+        // that returns `bis.SpatialViewDefinition` content instances which are related to their model selector
+        // through `bis.SpatialViewDefinitionUsesModelSelector` relationship by following it in backward
+        // direction (from `bis.SpatialViewDefinition` to `bis.ModelSelector`).
+        const ruleset: Ruleset = {
+          id: "example",
+          rules: [{
+            ruleType: RuleTypes.Content,
+            specifications: [{
+              specType: ContentSpecificationTypes.ContentInstancesOfSpecificClasses,
+              classes: { schemaName: "BisCore", classNames: ["ModelSelector"] },
+            }],
```

Comment:
I was thinking of using this example as a way to show that the SpatialViewDefinition instances are there because they are related to these ModelSelector instances through a relationship. Thoughts?

Reply:
I don't think that helps. IMO it's important that the comment explains we're passing a `ModelSelector` instance key as input when requesting content, and that we're getting a related `SpatialViewDefinition`.

Reply:
I like this example, but it would have been even better if it showcased how this rule handles one instance having multiple related instances with one-to-many relationship.